tests/test_emails.py

```diff
@@ -356,11 +367,196 @@ def test_send_email(self):
             msg = msg.format(localmachine=platform.node())
 
             mocked_send_email.assert_called_with(
+                attachments=None,
                 msg=msg,
                 subject='Samples Approved For Disposal',
                 mailhost='smtp.test.me',
                 port=25,
                 sender='sender@email.com',
                 recipients=['lab@email.com', 'project@email.com'],
                 strict=True
-            )
\ No newline at end of file
+            )
+
+class TestEmailContainerDispatched(TestEmailEPP):
+    def setUp(self):
+        super().setUp()
+        self.epp = self.create_epp(ContainerDispatchComplete)
+
+    def test_send_email(self):
+        with self.patch_project_single, self.patch_process, self.patch_samples, self.patch_email as mocked_send_email:
+            self.epp._run()
+            msg = '''Hi,
+
+The container dispatch has been completed for project1.
+
+https://{localmachine}/clarity/work-details/tep_uri
+
+Kind regards,
+ClarityX'''
+            msg = msg.format(localmachine=platform.node())
+
+            mocked_send_email.assert_called_with(
+                attachments=None,
+                msg=msg,
+                subject='project1: Container Ready For Dispatch',
+                mailhost='smtp.test.me',
+                port=25,
+                sender='sender@email.com',
+                recipients=['lab@email.com', 'project@email.com'],
+                strict=True
+            )
+
+    def test_send_email2(self):
+        with self.patch_project_multi, self.patch_process, self.patch_samples, self.patch_email as mocked_send_email:
+            with self.assertRaises(ValueError):
+                self.epp._run()
+
+class TestEmailContainerDispatched(TestEmailEPP):
```

Comment:
Class is defined twice

Reply:
Removed one instance, also removed test_send_email2 as did not seem to be applicable to script to be tested